src/main/java/AddList.java

```diff
@@ -0,0 +1,56 @@
+public class AddList {
+
+    private final String[] entries = new String[100]; 
+    private final boolean[] done = new boolean[100];   
```

Comment:
Great name! However, Boolean names should be made to sound like booleans (Eg. isDone)